packages/docs/src/components/layout/PageHeader.tsx

```diff
@@ -48,7 +48,7 @@ const PageHeader = ({ frontmatter = { title: '' }, theme }: PageHeaderProps) =>
           {storyId && (
             <a href={makeStorybookUrl(storyId, theme, 'docs')} className="c-page-header__link">
               <img
-                alt="Storybook logo"
+                alt=""
```

Comment:
These images are decorative only - so including `alt` text is just noise. Empty `alt` indicates a decorative image